Modules/.AL-Go/settings.json

```diff
@@ -14,6 +14,6 @@
                     ],
     "artifact":  "//21.1/base/first",
```

Comment:
Is this 21.1. is expected to remain 21.1?

Reply:
```suggestion
    "artifact":  "//21.3/base/first",
```